sharding/simple_sharding_problem.jl

```diff
@@ -49,7 +49,8 @@ function factors(N)
 end
 
 H = 8 # halo size
-T = Tx, Ty = 512 .* factors(ndevices)
+T = Tx, Ty = 512 .* factors(ndevices) 
+println("TTTTTTTTTTTTTTTTT: $(T)")
```

Comment:
```suggestion
@show T
```
but we already print the `model` CC @glwagner

Reply:
hmm, yeah i guess for some reason when we were playing around last night we either didn't see that or didn't realize it